docs/src/index.md

```diff
@@ -1,5 +1,6 @@
 ---
 layout: home
+markdownStyles: false
```

Comment:
This is the line that wont break the CSS in the new version :D they made a "fallback" for it to not override the custom css.